File: clientRest/src/main/java/formation/sopra/projet/clientRest/model/Formateur.java
package formation.sopra.projet.clientRest.model;

public class Formateur {
	private Integer id;
	private String prenom;
	private String nom;
	
	public Formateur() {
		
	}

	public Formateur(String prenom, String nom) {
		super();
		this.prenom = prenom;
		this.nom = nom;
	}

	public Integer getId() {
		return id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public String getPrenom() {
		return prenom;
	}

	public void setPrenom(String prenom) {
		this.prenom = prenom;
	}

	public String getNom() {
		return nom;
	}

	public void setNom(String nom) {
		this.nom = nom;
	}

	@Override
	public String toString() {
		return "Formateur [id=" + id + ", prenom=" + prenom + ", nom=" + nom + "]";
	}
	
	
	
}

File: formationAngular/src/app/model/formateur.ts
export class Formateur {
  public constructor(
    private _id?: number,
    private _prenom?: string,
    private _nom?: string
  ) {}

  public get id(): number {
    return this._id;
  }

  public set id(value: number) {
    this._id = value;
  }

  public get prenom(): string {
    return this._prenom;
  }

  public set prenom(value: string) {
    this._prenom = value;
  }
  public get nom(): string {
    return this._nom;
  }

  public set nom(value: string) {
    this._nom = value;
  }
}